client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -50,9 +66,21 @@ export const AuditAdminView: React.FC = () => {
   const [contests] = useContests(electionId, refreshId)
   const [auditSettings] = useAuditSettings(electionId, refreshId)
 
-  useEffect(() => {
-    refresh()
-  }, [refresh])
+  const [refreshTime, setRefreshTime] = useState(0)
+  const incrementRefreshTime = () => setRefreshTime(refreshTime + 1000)
+
+  // poll the apis every 5 minutes
+  useInterval(
+    () => {
+      setRefreshTime(0)
+      refresh()
+    },
+    300000,
+    true
+  )
+  // update the refresh time tracker every second
+  useInterval(incrementRefreshTime, 1000, false)
```

Comment:
how much do you trust JS intervals? in my experience, they are so-so at actually doing things on time over time. In other words, I don't trust that this refresh time will actually add up correctly. see: https://stackoverflow.com/a/8332678/1472662

I'm wondering if it would be more reliable to store the last refresh time as a timestamp and then have an interval that checks how long it's been. something like this:

```ts
const [lastRefreshTime, setLastRefreshTime] = useState(Date.now())
useInterval(() => {
  if (Date.now() - lastRefreshTime > 60 * 60 * 5) {
    refresh()
    setLastRefreshTime(Date.now())
  }
})

const refreshStatus = prettifyRefreshStatus(Date.now() - lastRefreshTime)
```

no idea if this works haha, just sketching

Reply:
@jonahkagan Good point, and yeah, I had some concerns about that as well. This does look like a more stable way of doing it. I'll see what I can put together!